Airline_Web_API/Airline_Web_API/Controllers/BookedTicketsController.cs

```diff
@@ -35,6 +35,20 @@ public async Task<ActionResult<BookedTicketViewModel>> GetBookedTicket(int id)
             return Ok(bookedTicket);
         }
 
+        [Authorize]
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<BookedTicketHistoryModel>>> GetBookedTicketsByUserId([FromQuery] int userId)
+        {
+            var bookedTickets = await _bookedTicketService.GetBookedTicketsByUserIdAsync(userId);
+
+            if (bookedTickets == null)
```

Comment:
Can it actually be null?

Reply:
Noooooo... I've just checked. I'll correct it